app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def link_to_add_row(name, f, association, **args)
```

Comment:
i'm not 100% sure I understand what this method is doing, but looks like this is a job for https://github.com/nathanvda/cocoon . I wonder whether this is not a piece of code straight from the `cocoon` itself :)

Reply:
@volodialeobit 
No, this snippet of code not from this gem (but from [here] ( https://www.driftingruby.com/episodes/nested-forms-from-scratch) ), but it is useful to know about gem which simplified implementation of nested forms.

Reply:
you better go with `cocoon` though, this way you won't be responsible for maintaining the code :)

Reply:
But is it compatible with rails 6?

Reply:
nice question! 
as I see, there may be some turbulence, but here is your solution in that case: https://github.com/nathanvda/cocoon/issues/555